articles/application-gateway/application-gateway-configure-ssl-policy-powershell.md

```diff
@@ -111,7 +111,16 @@ The following example sets a custom TLS policy on an application gateway. It set
 * TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
 
 > [!IMPORTANT]
-> TLS_RSA_WITH_AES_256_CBC_SHA256 must be selected when configuring a custom TLS policy. Application gateway uses this cipher suite for backend management. You can use this in combination with any other suites, but this one must be selected as well. 
+> When define a custom policy, enable at least one of the following cipher suites.
+> 
+> * TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+> * TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+> * TLS_DHE_RSA_WITH_AES_128_GCM_SHA256
+> * TLS_RSA_WITH_AES_128_GCM_SHA256
```

Comment:
> * TLS_RSA_WITH_AES_128_GCM_SHA256
> * TLS_RSA_WITH_AES_256_CBC_SHA256
> * TLS_RSA_WITH_AES_128_CBC_SHA256

These are definitely needed. After TLS 1.2 was enforced for Jarvis, I wonder if that requirement has changes the cipher suites.